src/components/Presentation/SlideNavigator/index.jsx

```diff
@@ -35,6 +40,21 @@ function SlideNavigator({ slides }) {
     },
   });
 
+  const updateSlideOrderMutation = useMutation(
```

Comment:
```suggestion
  const useUpdateSlideOrderMutation = useMutation(
```
커스텀 훅 네이밍 컨벤션에 따라 use를 붙여주시면 감사하겠습니다 🙇‍♂️